yarn-project/end-to-end/src/e2e_cross_chain_messaging.test.ts

```diff
@@ -27,24 +37,19 @@ describe('e2e_l1_to_l2_msg', () => {
   let tokenPortalAddress: EthAddress;
   let underlyingERC20Address: EthAddress;
   let rollupRegistryAddress: EthAddress;
-  let tokenPortal: GetContractReturnType<
-    typeof TokenPortalAbi,
-    PublicClient<HttpTransport, Chain>,
-    WalletClient<HttpTransport, Chain, Account>
-  >;
-  let underlyingERC20: GetContractReturnType<
-    typeof PortalERC20Abi,
-    PublicClient<HttpTransport, Chain>,
-    WalletClient<HttpTransport, Chain, Account>
-  >;
+  let tokenPortal: any;
```

Comment:
haha, fair enough making these any

Reply:
Linter started screaming and could not be bothered.